apps/web/src/client/components/file-uploader/_client-actions.js

```diff
@@ -196,17 +197,17 @@ const clientActions = (uploadForm) => {
 			buildProgressMessage({ show: true }, uploadForm);
 
 			let errors = null;
+			let uploadInfo;
 
 			if (fileList.length === 1 && uploadForm.dataset?.documentId) {
-				const uploadInfo = await getVersionUploadInfoFromInternalDB(fileList[0]);
-
-				errors = await uploadFiles(fileList, uploadInfo);
+				uploadInfo = await getVersionUploadInfoFromInternalDB(fileList[0]);
 			} else {
-				const uploadInfo = await getUploadInfoFromInternalDB(fileList);
-
-				errors = await uploadFiles(fileList, uploadInfo);
+				uploadInfo = await getUploadInfoFromInternalDB(fileList);
 			}
 
+			await relevantRepresentationsAttachmentUpload(uploadInfo, uploadForm);
```

Comment:
if I understand well, this function here is always executed. even for the uploads unrelated to represantion. am I wrong?